CONTRIBUTING.md

```
@@ -0,0 +1,66 @@
+```
+
+## Code Style
+
+- ESLint + Prettier for linting and formatting
+- Run `make lint` before committing
+
+## Commits
+
+Use [Conventional Commits](https://www.conventionalcommits.org/):
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)